### DISCUSSION TITLE
NumFOCUS affiliated vs. sponsored

### DISCUSSION THREAD

**Discussion by AdamRJensen** [pvlib/pvlib-python#1381] [General]
I've often wondered what the difference is between affiliated and sponsored NumFOCUS projects and why pvlib is in the affiliated category.

It seems that the benefits of being a NumFOCUS affiliated project are:
* participation in Google Summer of Code - example: #1289
* access to small development grants - example: #1333
* promotion of pvlib, e.g., numfocus website

Second, what concrete benefits would there be for pvlib to upgrade to a NumFOCUS sponsored project? And would we even be eligible or interested in this?

I suppose added administration might be a drawback of being a sponsored project, whereas access to funds to remove ads, pay for a domain name, etc. would be some of the benefits.

@mikofski I see that you are listed as an "Affiliated Project Selection Committee Member", so perhaps you are the right person to shed some light on this topic?

**Comment by mikofski** [comment]
NF explains the differences [here](https://numfocus.org/projects-overview):
> ### Fiscally Sponsored Projects
> Fiscally sponsored projects formally become part of NumFOCUS. NumFOCUS provides a number of services for fiscally sponsored projects, including financial administration, operational and legal support. Fiscally sponsored projects are also eligible to use NumFOCUS resources and infrastructure to help raise money for the project.
>
> Qualification for fiscal sponsorship requires meeting a higher bar than affiliation, to ensure that NumFOCUS can legally represent the project.

I believe the benefits are largely the same between sponsored vs. affiliated: 
* Community,
* Promotion, and
* Funding. (via GSoC and small dev grants)

But I think the major differences are:
* Sponsored projects are part of NF, receive admin & legal assistance, and therefore must pass a much higher bar for sponsorship
* Affiliated projects are independent of NF, handle their own admin/legal, and have a lower bar for affiliation.

@AdamRJensen I can say from my experience on the affiliated project selection committee that even for affiliated projects today, the bar is quite high. I would say pvlib meets the requirements for affiliated, but to be sponsored we would have to have a more transparent governance model, make all of our decisions public, have a more reliable release cadence, and demonstrate how pvlib is a critical component of the photovoltaic modeling community.

To counterpoint - what is pvlib lacking that we gain by NF sponsorship? Are we concerned about legal liability? Struggling to manage our administrative details and financial reporting obligations? Having trouble getting build servers? Planning events? Marketing? I see areas of growth for pvlib along some of these lines. One way to find out more is to join the [NF slack channel](http://numfocus.slack.com/) and ask questions on General.

Thanks for starting this thread. At the least it will get us thinking about where we're going as an organization, and possibly accelerate some processes like an open governance model and community organizing.

**Reply by AdamRJensen** [reply]
I guess financing a custom domain, e.g., pvlib.org or pvlib.io to host the documentation of the packages within the pvlib ecosystem is one thing that currently would be difficult. I suppose in general minor expenses, the first example that comes to mind was the recent discussion on paying to remove readthedocs adds as discussed [here](https://github.com/pvlib/pvlib-python/issues/1358#issuecomment-995918282).

**Comment by kandersolar** [comment]
Possibly an alternative: https://opencollective.com/
